src/main/java/se/lth/base/server/data/User.java

```diff
@@ -1,31 +1,96 @@
 package se.lth.base.server.data;
 
 import java.security.Principal;
+import java.sql.Timestamp;
 
 public class User implements Principal {
 
-    public static User NONE = new User(0, Role.NONE, "-");
+    public static User NONE = new User(0, Role.NONE, "-", "-", "-", "0", "-", 0, Timestamp.valueOf("2018-01-01 00:00:00"), false, 0, 0, "-");
 
-    private final int id;
+    private final int user_id;
     private final Role role;
     private final String username;
+    private final String first_name;
```

Comment:
camelCase for attribute names, goes for all underscores

Reply:
Min tanke var att det skulle bli tydligare om man hade samma namn i databasen som i koden men jag ändrar det.